Karumi/Shot
# AGP 7X alpha application id bug workaround

### :pushpin: References
* **Issue:** Workaround for #247

### :tophat: What is the goal?

Add a workaround for a bug introduced in AGP 7.0.1-alpha04 breaking Shot integration in android library modules.

### How is it being implemented?

We've just added an extension param where the user will have to configure the app id used for that module. This workaround will be removed in the future.

### How can it be tested?

I've added a new project as an E2E test to ensure the build is not broken anymore.
